### PR TITLE
Project status

### PR DESCRIPTION
**Pull request by joonsp**
- project colors similar to user log colors
- message for project status
- only on-going data, no history is preserved

screen shot 2017-09-20 at 10 26 06

### CODE REVIEW

File: server/src/api.js [hunk 1]
```diff
@@ -194,15 +194,20 @@ app.post(
   }),
 );
 
-app.post(
```
Comment (joonsp) [inline server/src/api.js]: this was a duplicate of

https://github.com/fraktio/teamdaily/pull/62/files#diff-75bb98844a56eb2824fd5f2b8fbb9c84L184

File: client/src/components/Button/index.js [hunk 2]
```diff
@@ -3,9 +3,10 @@ import cx from 'classnames';
 
 import styles from './style.pcss';
 
-const Button = ({ className, children, active, ...props }) => {
+const Button = ({ className, children, active, inactive, ...props }) => {
```
Comment (jaaniles) [inline client/src/components/Button/index.js]: Still needed?

File: client/src/components/Button/style.pcss [hunk 3]
```diff
@@ -32,3 +32,7 @@
 .active:after {
   content: ' \2713';
 }
+
+.inactive {
+  opacity: 0.3;
```
Comment (jaaniles) [inline client/src/components/Button/style.pcss]: Still needed?

File: client/src/components/ProjectView/ProjectBox.js [hunk 4]
```diff
@@ -1,26 +1,39 @@
...
-      <div className={styles.card}>
-        <h4 className={styles.projectTitle}>
-          {project.name}
-        </h4>
-        {project.employees.map(e => {
-          const entry = entries.filter(entry => entry.name === e.name);
-          const latestEntry = entry ? entry.get(-1) : null;
-          const color = latestEntry ? latestEntry.color : 'empty';
+      <div className={classnames(styles.outerCard)}>
+        <div className={classnames(styles.cahrdHeader, project.color)} />
```
Comment (jaaniles) [inline client/src/components/ProjectView/ProjectBox.js]: cahrdHeader -> cardHeader?

File: client/src/components/ProjectView/ProjectColor.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import classnames from 'classnames';
+import { injectIntl } from 'react-intl';
+import Select from 'react-select';
+
+import styles from './style.pcss';
+
+const COLOR_RED = 'red';
```
Comment (jaaniles) [inline client/src/components/ProjectView/ProjectColor.js]: Maybe move to somewhere else so can be used elsewhere? For example colors.js

File: client/src/components/ProjectView/ProjectColor.js [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+      </span>
+    </div>
+  );
+};
+
+const ProjectColor = props => {
+  const { color, saveProjectColor } = props;
+
+  const options = [
+    { value: COLOR_BLUE, label: 'Shred' },
```
Comment (jaaniles) [inline client/src/components/ProjectView/ProjectColor.js]: Consult and rethink if texts are good

File: client/src/components/ProjectView/ProjectColor.js [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+  };
+
+  return (
+    <Select
+      className="project-color-select"
+      name="project-color"
+      onChange={save}
+      optionRenderer={ProjectColorOption}
+      options={options}
+      placeholder="Status"
```
Comment (jaaniles) [inline client/src/components/ProjectView/ProjectColor.js]: Internationalize

File: client/src/colors.js [hunk 8]
```diff
@@ -1 +1,7 @@
-export default ['green', 'yellow', 'red', 'blue', 'pink'];
+export default [COLOR_GREEN, COLOR_YELLOW, COLOR_RED, COLOR_BLUE, COLOR_PINK];
+
+export const COLOR_RED = 'red';
+export const COLOR_YELLOW = 'yellow';
+export const COLOR_GREEN = 'green';
+export const COLOR_BLUE = 'blue';
+export const COLOR_PINK = 'blue';
```
Comment (jaaniles) [inline client/src/colors.js]: COLOR_PINK on purpose 'blue'?

File: client/src/components/ProjectView/ProjectColor.js [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+    },
+  ];
+
+  const selectedOption = options.findIndex(option => option.value === color);
+
+  const save = selected => {
+    selected ? saveProjectColor(selected.value) : saveProjectColor(null);
+  };
+
+  return (
```
Comment (jaaniles) [inline client/src/components/ProjectView/ProjectColor.js]: React select doesn't show properly when project is on the bottom of container

**Pull request merged**